### PR TITLE
update references to lio to fallback to jstag when not found

### PR DESCRIPTION
**Pull request by markhayden**

### CODE REVIEW

File: dist/pathfora.js [hunk 1]
```diff
@@ -3031,19 +3046,28 @@
...
-        if (
-          typeof window.lio.account.id === 'undefined' ||
-          window.lio.account.id === ''
-        ) {
-          throw new Error('Lytics Javascript tag returned an empty account id.');
-        }
-
-        pf.acctid = window.lio.account.id;
+        acctid = window.lio.account.id;
+      } else if (
```
Comment (sarahyablok) [inline dist/pathfora.js]: nit: can we get the comments here as well regarding legacy vs v3 tag?

File: dist/pathfora.js [hunk 2]
```diff
@@ -3624,9 +3648,18 @@
       // check for subfields if the value is an object
       var split = parts[0].trim().split('.');
 
-      dataval = window.lio.data;
-      var s;
+      // get entity data from tag
+      var dataval;
+      if (window.lio && window.lio.data) {
+        dataval = window.lio.data;
+      } else if (window.jstag && typeof window.jstag.getEntity === 'function') {
```
Comment (sarahyablok) [inline dist/pathfora.js]: same

File: src/rollup/display-conditions/entity-fields/replace-entity-field.js [hunk 3]
```diff
@@ -55,9 +55,18 @@ export default function replaceEntityField (
...
+    // get entity data from tag
+    var dataval;
+    if (window.lio && window.lio.data) {
+      dataval = window.lio.data;
+    } else if (window.jstag && typeof window.jstag.getEntity === 'function') {
+      var entity = window.jstag.getEntity();
+      if (entity && entity.data && entity.data.user) {
+        dataval = entity.data.user;
+      }
+    }
```
Comment (sarahyablok) [inline src/rollup/display-conditions/entity-fields/replace-entity-field.js]: notta big deal, but perhaps some of these repeated things sounds be functions. Like we could call a local `getEntity` function or something that handles the different tags.

File: test/acceptance/content-recommendation.spec.js [hunk 4]
```diff
@@ -458,7 +459,7 @@ describe('the content recommendation component', function () {
   it('should set the colors specified to the content recommendation modal', function (done) {
     window.lio = {
       account: {
-        id: 0
+        id: '123'
```
Comment (sarahyablok) [inline test/acceptance/content-recommendation.spec.js]: why did you change these?

File: test/acceptance/prioritized-widgets.spec.js [hunk 5]
```diff
@@ -177,7 +178,7 @@ describe('Prioritized widgets', function () {
         expect(widget2.length).toBe(0);
 
         expect(jasmine.Ajax.requests.mostRecent().url).toBe(
-          '//api.lytics.io/api/content/recommend/0/user/_uids/123?contentsegment=bb5ecbeadb9e572d66cd83d62d3dcd09'
+          '//api.lytics.io/api/content/recommend/123/user/_uids/123?contentsegment=bb5ecbeadb9e572d66cd83d62d3dcd09'
```
Comment (sarahyablok) [inline test/acceptance/prioritized-widgets.spec.js]: same here.

File: src/rollup/display-conditions/entity-fields/replace-entity-field.js (hunk 3 above)
Comment (markhayden) [reply src/rollup/display-conditions/entity-fields/replace-entity-field.js]: i actually want(ed) to refactor that entire file, its pretty cringy in spots but might be better as its own PR? opened this https://github.com/lytics/pathforajs/issues/316

File: test/acceptance/content-recommendation.spec.js (hunk 4 above)
Comment (markhayden) [reply test/acceptance/content-recommendation.spec.js]: because 0 is an invalid account

File: test/acceptance/prioritized-widgets.spec.js (hunk 5 above)
Comment (markhayden) [reply test/acceptance/prioritized-widgets.spec.js]: same as above

File: test/acceptance/content-recommendation.spec.js (hunk 4 above)
Comment (markhayden) [reply test/acceptance/content-recommendation.spec.js]: ooooh yah good call. did not realize that was the default. will update.

File: dist/pathfora.js (hunk 1 above)
Comment (markhayden) [reply dist/pathfora.js]: added to src (this is the dist)

File: dist/pathfora.js (hunk 2 above)
Comment (markhayden) [reply dist/pathfora.js]: added to src (this is the dist)

File: test/acceptance/prioritized-widgets.spec.js (hunk 5 above)
Comment (markhayden) [reply test/acceptance/prioritized-widgets.spec.js]: should be fixed
